### PR TITLE
Test facets

### PR DESCRIPTION
**Pull request by zzacharo**

### CODE REVIEW

File: tests/services/test_service.py [hunk 1]
```diff
@@ -104,3 +109,120 @@ def test_sorting_in_search(app, service, identity_simple, input_data):
 
     # Search with non existing sort parameter
     pytest.raises(ValidationError, service.search, identity_simple, sort="foo")
+
+
+def test_facets(app, service, identity_simple, input_data):
+    """Create a record."""
+    idx = 'records-record-v1.0.0'
```
Comment (lnielsen) [inline tests/services/test_service.py:116]: Minor: We smaller tests (in terms of lines of code). This will be hard to maintain long-term.

**Comment by ppanero** [comment]
Search has been migrated. Once `invenio-records-permissions` is released (https://github.com/inveniosoftware/invenio-records-permissions/pull/58) , tests would pass

**Comment by ppanero** [comment]
@lnielsen is good to go (Failing on ES6 same as master)

File: invenio_records_resources/services/records/service.py [hunk 2]
```diff
@@ -81,7 +85,18 @@ def create_search(self, identity, preference=True):
 
     def search_request(self, identity, params, preference=True):
         """Factory for creating a Search DSL instance."""
-        search = self.create_search(identity)
+        default_filter = permission_filter(permission=self.permission_policy(
```
Comment (ppanero) [inline invenio_records_resources/services/records/service.py:88]: @lnielsen search options

File: invenio_records_resources/services/records/service.py [hunk 3]
```diff
@@ -60,10 +61,13 @@ def record_cls(self):
         """Factory for creating a record class."""
         return self.config.record_cls
 
-    def create_search(self, identity, preference=True):
+    def create_search(self, identity, preference=True, search_options=None):
         """Instantiate a search class."""
-        search = self.config.search_cls(using=current_search_client)
-
+        _search_options = search_options if search_options else {}
```
Comment (ppanero) [inline invenio_records_resources/services/records/service.py:66]: @lnielsen search options

**Pull request closed without merging**